src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42FlowRttSpec.groovy

```diff
@@ -341,7 +341,10 @@ class Server42FlowRttSpec extends HealthCheckSpecification {
 
         when: "Disable flow rtt on dst switch"
         changeFlowRttSwitch(switchPair.dst, false)
-        sleep(1000)
+        Wrappers.wait(RULES_INSTALLATION_TIME, 3) {
+            northboundV2.validateSwitch(switchPair.dst.dpId).isAsExpected()
+        }
+        sleep(15000)
```

Comment:
1. Let's give this constant some meaningful name, like SERVER_42_RTT_STATS_LAG and put it into 'Constants' class.
2. Let's add this time to checkPointTime variable to not have mandatory wait in code.

Reply:
Done.